docs/net/index.md

```diff
@@ -0,0 +1,63 @@
+    content: Ethereum for .NET Developers
+  - property: og:description
+    content: Learn how to develop for Ethereum using .NET-based projects and tooling
+lang: en-US
+sidebar: auto
+sidebarDepth: 0
+---
+
+# Ethereum for .NET Developers
+
+<div class="featured">Learn how to develop for Ethereum using .NET-based projects and tooling</div><br>
+
+Use Ethereum to create decentralized applications (or "dapps") that utilize the benefits of cryptocurrency and blockchain technology. These dapps can be trustworthy, meaning that once they are deployed to Ethereum, they will always run as programmed. They can control digital assets in order to create new kinds of financial applications. They can be decentralized, meaning that no single entity or person controls them and are nearly impossible to censor.
+
+<img src="https://i.imgur.com/onFLM4L.png" width="100%" />
+
+## Getting Started with Smart Contracts and the Solidity Language
+
+**Take your first steps to integrate .NET with Ethereum**
+
+Need a more basic primer first? Check out [ethereum.org/learn](/learn/) or [ethereum.org/developers](/developers/).
+
+- [Blockchain Explained](https://kauri.io/article/d55684513211466da7f8cc03987607d5/blockchain-explained)
+- [Understanding Smart Contracts](https://kauri.io/article/e4f66c6079e74a4a9b532148d3158188/ethereum-101-part-5-the-smart-contract)
+- [Write your First Smart Contract](https://kauri.io/article/124b7db1d0cf4f47b414f8b13c9d66e2/remix-ide-your-first-smart-contract)
+- [Learn How to Compile and Deploy Solidity](https://kauri.io/article/973c5f54c4434bb1b0160cff8c695369/understanding-smart-contract-compilation-and-deployment)
+
+## Beginner articles
+
+- [Nethereum - An open source .NET integration library for blockchain](https://kauri.io/article/d15dfd4903f149cdb84b3ce666103b52/v1/nethereum-an-open-source-.net-integration-library-for-blockchain)
+- [Writing Ethereum Transactions to SQL Database Using Nethereum](https://medium.com/coinmonks/writing-ethereum-transactions-to-sql-database-using-nethereum-fd94e0e4fa36)
+- [See how to easily deploy Ethereum smart contracts using C# and VisualStudio](https://koukia.ca/deploy-ethereum-smart-contracts-using-c-and-visualstudio-5be188ae928c)
+- [A .NET Developer’s Workflow for Creating and Calling Ethereum Smart Contracts](https://medium.com/coinmonks/a-net-developers-workflow-for-creating-and-calling-ethereum-smart-contracts-44714f191db2)
+
+## Intermediate Articles
+
+- [Quick introduction to smart contracts integration with Nethereum](https://nethereum.readthedocs.io/en/latest/Nethereum.Workbooks/docs/nethereum-gettingstarted-smartcontracts/)
+- [Interfacing .NET and Ethereum Blockchain Smart Contracts with Nethereum](https://medium.com/singapore-blockchain-dapps/interfacing-net-and-ethereum-blockchain-smart-contracts-with-nethereum-7b46b71f2dae)
+- [https://tech-mint.com/create-asp-net-core-web-api-for-ethereum-dapps/](https://tech-mint.com/create-asp-net-core-web-api-for-ethereum-dapps/)
```

Comment:
Please provide description text here vs. the raw URL

Reply:
Oopsie, missed this